okhttp-tls/src/main/java/okhttp3/tls/HandshakeCertificates.kt

```diff
@@ -124,10 +153,20 @@ class HandshakeCertificates private constructor(
      * certificates. Applications that connect to a known set of servers may be able to mitigate
      * this problem with [certificate pinning][CertificatePinner].
      */
-    fun addPlatformTrustedCertificates() = apply {
-      val platformTrustManager = Platform.get().platformTrustManager()
-      Collections.addAll(trustedCertificates, *platformTrustManager.acceptedIssuers)
-    }
+    @get:JvmName("addPlatformTrustedCertificates") val addPlatformTrustedCertificates: Builder
+      get() {
+        return apply {
+          val platformTrustManager = Platform.get().platformTrustManager()
+          Collections.addAll(trustedCertificates, *platformTrustManager.acceptedIssuers)
+        }
+      }
+
+    @JvmName("-deprecated_sslSocketFactory")
+    @Deprecated(
+        message = "moved to val",
+        replaceWith = ReplaceWith(expression = "sslSocketFactory"),
+        level = DeprecationLevel.WARNING)
+    fun addPlatformTrustedCertificates(): Builder = addPlatformTrustedCertificates
```

Comment:
Should we apply idiomatic Kotlin to Builder methods as well? 👀 My question could be: can we do an idiomatic Kotlin way like the following without `()` for Builder methods? 🤔 
```
Builder
  .computeA
  .build
```

Reply:
No, definitely not

Reply:
The goal isn’t to remove the parens, it’s to use `vals` where the semantics of them work.
https://kotlinlang.org/docs/reference/properties.html